components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -203,7 +185,6 @@ impl From<nsCSSPropertyID> for TransitionProperty {
                 TransitionProperty::Shorthand(ShorthandId::${prop.camel_case})
             }
             % endfor
-            nsCSSPropertyID::eCSSPropertyExtra_all_properties => TransitionProperty::All,
```

Comment:
I suspect, unless this is dead code, that this will need Gecko changes.

In particular, Gecko checks for `eCSSPropertyExtra_all_properties` instead of `eCSSProperty_all`... cc @hiikezoe

Reply:
Ah ah, I kept the loop on `data.shorthands_except_all()` just above, so this will just panic in Gecko…